### PR TITLE
Mileston 1/feat scrum 43 admin feedback backend

### PR DESCRIPTION
**Pull request by OmniaTarek18**
### Admin Feedback Backend

1. add feedback entity 
2. get different entities from other branches
3. add relationships between different entities and feedback
4. implement feedback service 
5. implement feedback dynamic filtering using specification
6. implement feedback endpoints
7. implement DTOs and DTO Mappers

### CODE REVIEW

File: backend/src/main/java/com/example/backend/Services/FeedbackDisplayService.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.springframework.data.domain.Sort;
+import org.springframework.data.jpa.domain.Specification;
+
+@RequiredArgsConstructor
+@Service
+public class FeedbackDisplayService {
+
+    private final FeedbackRepository feedbackRepository;
+
+    public short getAvergaeRating(){
```
Comment (Rawan-M-Said) [inline backend/src/main/java/com/example/backend/Services/FeedbackDisplayService.java]: Rename the function to be "getAverageRating" (There is a typo)

File: backend/src/main/java/com/example/backend/Controllers/AdminDashboard/FlightDislplayController.java
Comment (Rawan-M-Said) [inline backend/src/main/java/com/example/backend/Controllers/AdminDashboard/FlightDislplayController.java:1]: Rename the file to be "FlightDisplayController.java" (There is a typo)

File: backend/src/main/java/com/example/backend/Services/FeedbackDisplayService.java (hunk 1 above)
Comment (OmniaTarek18) [reply backend/src/main/java/com/example/backend/Services/FeedbackDisplayService.java]: done

File: backend/src/main/java/com/example/backend/Controllers/AdminDashboard/FlightDislplayController.java
Comment (OmniaTarek18) [reply backend/src/main/java/com/example/backend/Controllers/AdminDashboard/FlightDislplayController.java:1]: done

File: backend/src/main/java/com/example/backend/Controllers/AdminDashboard/FlightDislplayController.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@Controller
+@RequestMapping("/admin")
+public class FlightDislplayController {
+
+    private final FlightDisplayService flightService;
+
+    public FlightDislplayController(FlightDisplayService flightService) {
```
Comment (Mohamed-Hammady) [inline backend/src/main/java/com/example/backend/Controllers/AdminDashboard/FlightDislplayController.java]: typo

Comment (Mohamed-Hammady) [inline backend/src/main/java/com/example/backend/Controllers/AdminDashboard/FlightDislplayController.java]: typo

**Pull request merged**